src/components/decoration/textBlocks.tsx

```diff
@@ -35,6 +35,7 @@ interface SectionHeaderProps
   stackEntries?: boolean;
   className?: string;
   rootClassName?: string;
+  smallOnMobile?: boolean;
```

Comment:
Does it make sense to change the text size on mobil for every use of this component? We should keep the text sizes consistent over all pages.

Reply:
The layout doesn't work on mobile without smaller headings
![image](https://github.com/user-attachments/assets/9d6be948-afa5-4877-8f41-5a312170768c)

Reply:
I like the updated style for mobile, I was just curious if it makes sense to update the style not only for this yir page but also for all other places where this component is used. This would ensure a consistent look and help reduce the number of props on this component.

Reply:
The component has 117 references so it will be tricky to see if using the smaller font globally is going to cause issues in any of those circumstances. I'll come back to it if I get time.

Reply:
Should we track this on our board then?
I don't think much testing will be necessary for this. Checking it in a few places should be enough.